NeuroID/src/main/java/com/neuroid/tracker/utils/NIDTimerActive.kt

```diff
@@ -31,6 +35,10 @@ object NIDTimerActive {
 
     @Synchronized fun restartTimerActive() {
         timer.cancel()
+        if (!timer.isSensorRunning) {
```

Comment:
```suggestion
        if (!NIDSensorHelper.isSensorRunning) {
```

Reply:
Or even just make `NIDSensorHelper.restartSensors()` safe to call when running...